Add ability to use use sub-rule matches on skip or warn lists (#2251)
ansible/ansible-lint: src/ansiblelint/rules/yaml.md

You can fully disable all yamllint violations by adding `yaml` to the `skip_list`.

Specific tag identifiers that are printed at the end of the rule name,
like `yaml[trailing-spaces]` or `yaml[indentation]` can also be skipped, allowing
you to have more control.

> Keep in mind that our tool does not take into consideration the warning level
> of yamllint and we consider all matches as errors. Still, if you want to treat
> some of these as warnings, you can add them to `warn_list`.

Reviewer: ```suggestion
Keep in mind that `ansible-lint` does not take into consideration the warning level
of yamllint; we treat all yamllint matches as errors. So, if you want to treat
some of these as warnings, add them to `warn_list`.
```
